packages/openapi-request-coercer/test/data-driven/not-throw-on-untyped-array-params-in-request.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  logs: [
+    {
+      type: 'WARN',
+      message: '',
+      optionalParams: [
+        "No type has been defined. A default 'identity' strategy has been set."
```

Comment:
@jsdevel Although that's a bit better than before, it still may not be completely user friendly.

Given that log message and a very long spec, it could be very difficult/cumbersome for the user to actually dig out which type is missing.

A proposal could be to pass down the `param` being dealt with and add it to this message (in a stringified Json format) to help with some context.

Thoughts?

Reply:
i think it's fine.  is this ready to merge?

Reply:
> i think it's fine. is this ready to merge?

@jsdevel I believe it is.